{"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3583", "repository_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT", "labels_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3583/labels{/name}", "comments_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3583/comments", "events_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3583/events", "html_url": "https://github.com/Significant-Gravitas/Auto-GPT/issues/3583", "id": 1689967949, "node_id": "I_kwDOJKSTjM5kut1N", "number": 3583, "title": "Results are not written to a file (disappointing ongoing issue)", "user": {"login": "concenit", "id": 56778787, "node_id": "MDQ6VXNlcjU2Nzc4Nzg3", "avatar_url": "https://avatars.githubusercontent.com/u/56778787?v=4", "gravatar_id": "", "url": "https://api.github.com/users/concenit", "html_url": "https://github.com/concenit", "followers_url": "https://api.github.com/users/concenit/followers", "following_url": "https://api.github.com/users/concenit/following{/other_user}", "gists_url": "https://api.github.com/users/concenit/gists{/gist_id}", "starred_url": "https://api.github.com/users/concenit/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/concenit/subscriptions", "organizations_url": "https://api.github.com/users/concenit/orgs", "repos_url": "https://api.github.com/users/concenit/repos", "events_url": "https://api.github.com/users/concenit/events{/privacy}", "received_events_url": "https://api.github.com/users/concenit/received_events", "type": "User", "site_admin": false}, "labels": [{"id": 5451094099, "node_id": "LA_kwDOJKSTjM8AAAABROkYUw", "url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/labels/function:%20fileops", "name": "function: fileops", "color": "fef2c0", "default": false, "description": ""}], "state": "open", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 5, "created_at": "2023-04-30T15:45:46Z", "updated_at": "2023-05-02T02:47:18Z", "closed_at": null, "author_association": "NONE", "active_lock_reason": null, "body": "### \u26a0\ufe0f Search for existing issues first \u26a0\ufe0f\n\n- [X] I have searched the existing issues, and there is no existing issue for my problem\n\n### Which Operating System are you using?\n\nWindows\n\n### Which version of Auto-GPT are you using?\n\nLatest Release\n\n### GPT-3 or GPT-4?\n\nGPT-3.5\n\n### Steps to reproduce \ud83d\udd79\n\nAsk Auto-GPT to perform a task and write to results to a file. The program performs the task perfectly, but doesn\u2019t provide a file containing the results (although it reports that it has done so).\r\n\r\n\n\n### Current behavior \ud83d\ude2f\n\n1.\tAuto-GPT warns \u201cWarning: The file 'auto-gpt.json' does not exist. Local memory would not be saved to a file.\u201d\r\n2.\tAuto-GPT reports writing to a file, but the file can\u2019t be found anywhere\n\n### Expected behavior \ud83e\udd14\n\nWrite the results to a file\n\n### Your prompt \ud83d\udcdd\n\n Write to a file the name of the capital of the state of Montana\r\n\n\n### Your Logs \ud83d\udcd2\n\n```log\r\n<insert your logs here>\r\n```\r\n", "reactions": {"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3583/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3583/timeline", "performed_via_github_app": null, "state_reason": null}